### PR TITLE
Awinterman

### PR DESCRIPTION
**Pull request by AWinterman**
So this is the mini-est profile, but so it goes.

### CODE REVIEW

File: templates/andrew-winterman.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Your Name Here
```
Comment (phated) [inline templates/andrew-winterman.md]: Need your name

File: templates/andrew-winterman.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+![an avatar, if you'd like to include
```
Comment (phated) [inline templates/andrew-winterman.md]: And avatar

Comment (AWinterman) [inline templates/andrew-winterman.md]: That is my avatar (click the link)

Comment (phated) [inline templates/andrew-winterman.md]: Sorry, meant alt text

Comment (AWinterman) [inline templates/andrew-winterman.md]: You want me it change the alt text?

**Comment by hackygolucky** [comment]
:+1:

**Comment by chrisdickinson** [comment]
@AWinterman ah -- could you move this to the `portland-or-usa` directory?

**Pull request closed without merging**